doc/designs/ldap_grace_period.md

```diff
@@ -51,7 +51,20 @@ The basic flow is:
 
 On successful password reset (by anyone) reset the user's passwordGraceUserTime to 0.
 
-The default value on install/upgrade will be -1 to retail existing behavior.
+Range values for passwordgracelimit are:
+
+-1 : password grace checking is disabled
+ 0 : no grace BIND are allowed at all post-expiration
+ 1..MAXINT: the number of BIND allowed post-expiration
+
+The default value on install/upgrade will be -1 to retain existing behavior.
+
+Existing group policies are not updated to reflect the new behavior. Lack
+of a value in LDAP is treated as 0 so any existing group policies will not
+allow post-expiration BIND.
```

Comment:
I would remove the first sentence `Existing group policies are not updated to reflect the new behavior.` because it doesn't reflect that the upgrade adds the default value = -1 to existing group policies without passwordgracelimit attribute.

Reply:
Sigh. Yeah, I wrote the design first and missed that we had agreed to update all existing group policy to -1. Then I added a patch to do exactly that.

I fixed the design and re-ordered the new paragraphs a bit into I think a more clear ordering.